src/SFA.DAS.EmployerFinance.Messages/Events/LevyAddedToAccountEvent.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace SFA.DAS.EmployerFinance.Messages.Events
 {
-    public class LevyAddedToAccount : Event
+    public class LevyAddedToAccountEvent : Event
```

Comment:
Due to how subscriptions currently work I think we should leave the name of this event, as consumers expect the old name and namespace. I think all consumers would have to update in order to consume these events